Graphics/Master Graphics Installer.event

```diff
@@ -22,6 +22,16 @@ ORG 0x5926F4+(0x80*PiercingBowIconID)
 #incext Png2Dmp "PiercingBow.png"
 ORG 0x5926F4+(0x80*HuntersBowIconID)
 #incext Png2Dmp "HuntersBow.png"
+
+/* TODO pending: IIDs for these items
```

Comment:
You can just use the empty rows after RunicBow in the csv.

Reply:
I see you've added them to the item table. Please remove the TODO note then.

Reply:
Icon ID, not Item ID.